<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maze</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>500</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="load_maze">
         <property name="text">
          <string>Загрузить лабиринт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_maze">
         <property name="text">
          <string>Сохранить лабиринт</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="gen_maze">
         <property name="text">
          <string>Генерировать лабирин</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="size_maze_text">
           <property name="text">
            <string>Размер(x,y):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="size_maze_x">
           <property name="minimum">
            <number>4</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="size_maze_y">
           <property name="minimum">
            <number>4</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="find_solving_maze">
         <property name="text">
          <string>Найти решение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="start_point_text">
         <property name="text">
          <string>Начало (x,y):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="start_x">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="start_y">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="end_point_text">
         <property name="text">
          <string>Конец (x,y):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="end_x">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="end_y">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="cave_window">
       <property name="text">
        <string>Открыть окно создания пещер</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
